TokenSale/test/blockchainLabsNZ/burnable_tests.js

```diff
@@ -0,0 +1,118 @@
+let KNC = artifacts.require("KyberNetworkCrystal");
+
+const assertFail = require("./helpers/assertFail");
+const BigNumber = require("bignumber.js");
+
+let knc;
+
+let owner;
+
+// Taken from Migrations
+Date.prototype.getUnixTime = function() { return this.getTime()/1000|0 };
+if(!Date.now) Date.now = function() { return new Date(); }
+Date.time = function() { return Date.now().getUnixTime(); }
+
+
+contract("KNC", function(accounts) {
+  beforeEach(async () => {
+
+    owner = accounts[0];
+    var publicSaleStartTime = new Date('Sat, 16 Sep 2017 06:00:00 GMT').getUnixTime();
+    var publicSaleEndTime = new Date('Sun, 24 Sep 2017 06:00:00 GMT').getUnixTime();
+
+    knc = await KNC.new(
+        226000000000000000000000000, //Initial supply
```

Comment:
maybe define:

`const initial_supply = 226000000000000000000000000`

at the beginning of the file, so you keep it DRY?

Reply:
DRY code, DAMP tests.
but it's just a suggestion, not a rule.

https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests

Reply:
@Janther #TIL

Reply:
you should still declare it with `let`